File: JULY-2025/21-07-2025/p1957.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        res = ""
        count = 1
        
        for i in range(len(s)):
            if i > 0 and s[i] == s[i - 1]:
                count += 1
            else:
                count = 1

            if count <= 2:
                res += s[i]
        
        return res

File: JULY-2025/21-07-2025/p1957.dart
class Solution {
  String makeFancyString(String s) {
    List<String> str=s.split('');
    String char=str[0];
    int count=0;
    List<String> r=[];
    for(int i=0;i<str.length;i++){
        if(str[i]==char){
          count++;
        }else{
            char=str[i];
            count=1;
        }
        if(count==3){
           // str.removeAt(i);
           //i--;
            count--;
        }else{
            r.add(str[i]);
        }
    }return r.join();
  }
}
